aspnetcore/mvc/views/razor.md

```diff
@@ -60,6 +60,14 @@ With the exception of the C# `await` keyword, implicit expressions must not cont
 <p>@await DoSomething("hello", "world")</p>
 ```
 
+Implicit expressions may not contain C# generics, as the characters inside the brackets (`<>`) would be interpreted as an HTML tag:
+
+```cshtml
+<p>@GenericMethod<int>()</p>
+```
+
+The code generates a compiler error like `The "int" element was not closed.  All elements must be either self-closing or have a matching end tag` or `Cannot convert method group 'GenericMethod' to non-delegate type 'object'. Did you intend to invoke the method?`  Instead, generic method calls must be wrapped in an [explicit Razor expression](#explicit-razor-expressions) or a [Razor code block](#razor-code-blocks).  Note that this restriction does not apply to _.vbhtml_ Razor files, because Visual Basic syntax places parentheses around generic type parameters instead of brackets.
```

Comment:
* Don't sweat the double-space inside the error message ("closed.  All") ... that's fine; but elsewhere go with one space between sentences (e.g., "method?\`  Instead", "blocks).  Note")
* does not :point_right: doesn't
* Underscores work, but we go with asterisks by convention: \_.vbhtml\_ :point_right: \*.vbhtml\*
* Remove comma: "Razor files, because" :point_right: "Razor files because"

Reply:
👍